docs/layers.md

```diff
@@ -399,7 +399,7 @@ Places with `kind` values of `continent`, `country`, with others added starting
 #### Place properties (common optional):
 
 * `capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-0 capital`, `Admin-0 capital alt`, and `Admin-0 region capital`.
-* `state_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
+* `region_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
```

Comment:
Looks like this was addressed in https://github.com/tilezen/vector-datasource/pull/942/commits/8b415af33acfb320baba7775696905ed7a0b7baa and https://github.com/tilezen/vector-datasource/pull/942/commits/1b577aee809e04564dd130788fcff5c7842819e9.